ws-android/Dockerfile

```diff
@@ -27,43 +38,50 @@ RUN SDK_VERSION=r24.3.3 && \
     bash -c 'for file in $(find /usr/local/android-sdk-*); do [[ -x $file ]] && chmod +x "$file"; done'; \
     exit 0
 
-# Install Android NDK
-# https://developer.android.com/tools/sdk/ndk/index.html
-# https://developer.android.com/ndk/index.html
-RUN NDK_VERSION=r10e && \
-    cd /usr/local && \
-    curl -L -o android-ndk.bin http://dl.google.com/android/ndk/android-ndk-${NDK_VERSION}-linux-x86_64.bin && \
-    chmod a+x android-ndk.bin && \
-    ./android-ndk.bin && \
-    rm -f android-ndk.bin && \
-    chmod -R +r /usr/local/android-ndk-* && \
-    bash -c 'for file in $(find /usr/local/android-ndk-*); do [[ -x $file ]] && chmod +x "$file"; done'
-
-# Install Gradle
-RUN GRADLE_VERSION=2.5 && \
-    cd /usr/local && \
+# Android SDK Home and Path for pre-cloud9 testing
+ENV ANDROID_SDK_HOME /usr/local/android-sdk-linux
+ENV ANDROID_HOME /usr/local/android-sdk-linux
+ENV PATH $PATH:$ANDROID_SDK_HOME
+ENV PATH $PATH:$ANDROID_SDK_HOME/tools
+ENV PATH $PATH:$ANDROID_SDK_HOME/platform-tools
+
+# -------------------------- Install the Android Platforms -----------------------------------------
+
+# This fancy "for" loop solves an install issue
+RUN for filter in tools platform-tools android-20 android-22 android-23 extra build-tools-23.0.2; do \
```

Comment:
Lets just install tools, platform-tools, extra, build-tools and the latest android version here, people can install older ones if they like afterwards.